package.json

```diff
@@ -70,6 +70,7 @@
     "lodash-webpack-plugin": "^0.3.0",
     "mocha": "^2.4.5",
     "react": "^15.0.0",
+    "react-addons-shallow-compare": "^15.0.0",
```

Comment:
This is critical. If you don't add it to `dependencies` or `peerDependencies`, the package will be broken.